### PR TITLE
Files comparison localfiles datasets

### PR DESCRIPTION
**Pull request by sarthakjdev**
This PR resolves [Issue#1444](https://github.com/zowe/zowe-cli/issues/1444).

Added a command ```zos-files compare local-file-data-set``` to compare the differences between a local file and a dataset.

Sample command: ```zowe files cmp lds "./package.json" "mentee5.public.jcl(test)"  ``` 

It will take two arguments first will be the local file path either relative or absolute and the second one is the data-set name.

--bv options can be used to get the same differences in the browser.

### CODE REVIEW

File: packages/workflows/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe z/OS workflows SDK package will be documented in this file.
 
+## Recent Changes
```
Comment (gejohnston) [inline packages/workflows/CHANGELOG.md]: Should the changelog be updated in zosfiles package instead of the workflows package?

**Review by gejohnston — CHANGES_REQUESTED** [review]

File: packages/workflows/CHANGELOG.md (hunk 1 above)
Comment (sarthakjdev) [reply packages/workflows/CHANGELOG.md]: Hey @gejohnston , 
I have rectified this thing, pls have a look at the updated one.

**Comment by zFernand0** [comment]
Seems like we might be missing some coverage for the handler itself.

Since we plan to combine all Compare handlers into a common CompareBaseHandler in the near future, that's why I'm not too worries about it 😋

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋

**Review by gejohnston — APPROVED** [review]

File: packages/cli/src/zosfiles/compare/lds/LocalfileDataset.definition.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+// Does not use the import in anticipation of some internationalization work to be done later.
+const strings = (require("../../-strings-/en").default as typeof i18nTypings).COMPARE;
+
+/**
+ * Compare local files and data sets command definition containing its description, examples and/or options
+ * @type {ICommandDefinition}
+ */
+export const LocalfileDatasetDefinition: ICommandDefinition = {
+    name: "local-file-data-set",
+    aliases: ["lds"],
```
Comment (t1m0thyj) [inline packages/cli/src/zosfiles/compare/lds/LocalfileDataset.definition.ts:25]: Should the alias be `lfds`?

Comment (sarthakjdev) [reply packages/cli/src/zosfiles/compare/lds/LocalfileDataset.definition.ts:25]: yes, `lfds` sounds more defined for the command let me change it. 
Thanks!

**Pull request merged**